vagrant/README.md

```diff
@@ -124,3 +124,11 @@ Now start things up, but specify the aws provider:
 Your instances should get tagged with a name including your hostname to make
 them identifiable and make it easier to track instances in the AWS management
 console.
+
+### Libvirt with kvm ###
+
+Install the `vagrant-libvirt` plugin to provide libvirt and kvm support:
+```vagrant plugin install vagrant-libvirt``` or ```yum install vagrant-libvirt``` for fedora >=21
```

Comment:
Link seems better -- no need to duplicate the info. Maybe just add that this provider was tested with a Fedora host system? Between different distros, different JDKs, etc. we're probably not going to be able to test every single configuration.